pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -99,6 +100,8 @@ var (
 		"egressNetworkPolicyRuleAction",
 		"tcpState",
 		"flowType",
+		"egressPolicyName",
```

Comment:
I may have used the term "egress policy" in the past, but I don't think it is a standard term we use in Antrea (code / documentation). It may be better to just refer to this as "egressName" to avoid confusion with egress NP rules.